packages/vue/src/components/organisms/SfHero/_internal/SfHeroItem.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <li class="glide__slide sf-hero-item" :style="style" data-testid="hero-item">
-    <component :is="wrapper" class="sf-hero-item__wrapper" :link="link">
+    <div
+      class="sf-hero-item__wrapper sf-hero-item__wrapper--desktop"
+      :link="link"
```

Comment:
I think we can remove this link on div :)

Reply:
done :)